include/aspect/particle/output/hdf5.h

```diff
@@ -24,7 +24,7 @@
 #include <aspect/particle/output/interface.h>
 #include <aspect/simulator_access.h>
 
-#include <deal.II/base/data_out_base.h>
+#include <deal.II/numerics/data_out.h>
```

Comment:
not wrong, but why do you need this change? it pulls in a lot of stuff you don't immediately need.